keyboard/src/androidLibMain/kotlin/KeyboardManager.kt

```diff
@@ -60,7 +60,7 @@ actual class KeyboardManager(
 
     override fun show(keyboardHostingView: KeyboardHostingView) {
         inputMethodManager?.let {
-            activity?.findViewById<View>(keyboardHostingView)?.requestFocus()
```

Comment:
What if I still want to use the old UI tookit?

Reply:
Good point, I made the PR a draft because of this, sorry for not mentioning it in the description. The solution in my opinion would be to create a separate KeyboardManager for compose, but that would cause duplication of code especially in the ios part because there isn't actually a swift UI way to focus on a TextField, but rather a UIKit wrapper for the view.

Second possible solution would be to make KeyboardHostingView a sealed class that has 2 subclasses: `AndroidView(id: Int)` and `ComposeView(focusRequester: FocusRequester)` then from `show` method a `KeyboardHostingView` is expected with a when we can do:
```
when(keyboardHostingView) {
    is ComposeView -> keyboardHostingView.focusRequester.requestFocus()
    is AndroidView -> activity?.findViewById<View>(keyboardHostingView.id)?.requestFocus()
}
```

What do you think?

Reply:
I'd rather suggest to change `KeyboardHostingView` type - since it's main purpose is to help to request a focus, it should be transformed into a `FocusRequester` interface with a single `requestFocus()` method, then we can have a separate implementation of that interface per platform/UI toolkit.

Reply:
1d9af66 0d872d8 c063c1e